rad/src/bin/git-remote-rad.rs

```diff
@@ -0,0 +1,52 @@
+use git_remote_rad::{run, Options};
+use radicle_common::args::Args;
+use radicle_common::profile;
+
```

Comment:
I don't think this file should be inside `rad`, since it's not the name of the command.

Reply:
I would expect it to be in the `git-remote-rad` package I guess